### PR TITLE
fix: CORE-1509 - Docker image building for EE failing due to alpine-p…

### PR DESCRIPTION
**Pull request by mikhaelsantos**
* Update alpine-pkg-glibc from 2..27 -> 2.28
* Fix Public signing key location

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ RUN apk add --no-cache \
             && apk add --no-cache --virtual .build-deps \
                 binutils-gold \
                 curl \
+                wget \
```
Comment (kare) [inline Dockerfile]: Just use curl instead of wget since its already there

File: Dockerfile [hunk 2]
```diff
@@ -19,9 +20,12 @@ RUN apk add --no-cache \
                 make \
                 python
 
-RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub
-RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.27-r0/glibc-2.27-r0.apk
-RUN apk add glibc-2.27-r0.apk
+#
+ENV ALPINE_PKG_GLIBC_VERSION 2.28-r0
+
+RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
```
Comment (kare) [inline Dockerfile]: sgerrand.com tls certificate has expired. Any other alternatives?

Comment (kare) [reply Dockerfile]: So the certificate for https://alpine-pkgs.sgerrand.com is ok, but not ok for sgerrand.com

Comment (mikhaelsantos) [reply Dockerfile]: Checked https://alpine-pkgs.sgerrand.com which is ok. Pinged developer. Lets keep it for now.

**Pull request merged**